src/main/java/frc/trigon/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,138 @@
+                },
+                () -> false,
+                this
+        );
+    }
+
+    private void generateAngleMotorProfile(Rotation2d targetAngle) {
+        angleMotorProfile = new TrapezoidProfile(
+                ArmConstants.ANGLE_CONSTRAINTS,
+                new TrapezoidProfile.State(targetAngle.getDegrees(), 0),
+                new TrapezoidProfile.State(getAngleMotorPositionDegrees().getDegrees(), getAngleMotorVelocityDegreesPerSecond())
+        );
+        lastAngleMotorProfileGenerationTime = Timer.getFPGATimestamp();
+    }
+
+    private void generateElevatorMotorProfile(double targetElevatorPosition) {
+        elevatorMotorProfile = new TrapezoidProfile(
+                ArmConstants.ELEVATOR_CONSTRAINTS,
+                new TrapezoidProfile.State(targetElevatorPosition, 0),
+                new TrapezoidProfile.State(getElevatorMotorPositionRevolutions(), getElevatorMotorVelocityRevolutions())
+        );
+        lastElevatorMotorProfileGenerationTime = Timer.getFPGATimestamp();
+    }
+
+    private void setTargetAngleFromProfile() {
+        if (angleMotorProfile == null) {
+            stopAngleMotors();
+            return;
+        }
+        final TrapezoidProfile.State targetState = angleMotorProfile.calculate(getAngleMotorProfileTime());
+        masterAngleMotor.getPIDController().setP(ArmConstants.ANGLE_PID_CONTROLLER.getP());
+        masterAngleMotor.getPIDController().setI(ArmConstants.ANGLE_PID_CONTROLLER.getI());
+        masterAngleMotor.getPIDController().setD(ArmConstants.ANGLE_PID_CONTROLLER.getD());
+        masterAngleMotor.getPIDController().setReference(targetState.position, CANSparkMax.ControlType.kPosition);
+    }
+
+    private void setTargetPositionFromProfile() {
+        if (elevatorMotorProfile == null) {
+            stopElevatorMotors();
+            return;
+        }
+        final TrapezoidProfile.State targetState = elevatorMotorProfile.calculate(getElevatorMotorProfileTime());
+        masterElevatorMotor.getPIDController().setP(ArmConstants.ELEVATOR_PID_CONTROLLER.getP());
+        masterElevatorMotor.getPIDController().setI(ArmConstants.ELEVATOR_PID_CONTROLLER.getI());
+        masterElevatorMotor.getPIDController().setD(ArmConstants.ELEVATOR_PID_CONTROLLER.getD());
+        masterElevatorMotor.getPIDController().setReference(targetState.position, CANSparkMax.ControlType.kPosition);
+    }
+
+    private double getAngleMotorProfileTime() {
+        return Timer.getFPGATimestamp() - lastAngleMotorProfileGenerationTime;
+    }
+
+    private double getElevatorMotorProfileTime() {
+        return Timer.getFPGATimestamp() - lastElevatorMotorProfileGenerationTime;
+    }
+
+    private Rotation2d getAngleMotorPositionDegrees() {
+        return Rotation2d.fromRotations(angleEncoder.getPosition().getValue());
+    }
+
+    private double getElevatorMotorPositionRevolutions() {
+        return magTicksToRevolutions(elevatorEncoder.getSelectedSensorPosition());
```

Comment:
Never ever ever use static imports. Use `ClassName.method` (In this case `Conversion.magTicksToRevolutions`)